src/core_plugins/kibana/public/dashboard/listing/dashboard_listing.js

```diff
@@ -282,15 +282,13 @@ export class DashboardListing extends React.Component {
         name: 'Title',
         sortable: true,
         render: (field, record) => (
-          <span>
-            <EuiLink
-              className="dashboardLink"
-              href={`#${createDashboardEditUrl(record.id)}`}
-              data-test-subj={`dashboardListingTitleLink-${record.title.split(' ').join('-')}`}
-            >
-              {field}
-            </EuiLink>
-          </span>
+          <EuiLink
```

Comment:
`<span>` wrapper needed because of EUI bug https://github.com/elastic/eui/issues/706 which has been fixed.